### DISCUSSION TITLE
artifact rolls

### DISCUSSION THREAD

**Discussion by maxkleehu** [Melledy/Grasscutter#1362] [Q&A] [answered]
How can I get rid of min substat roll as a possible drop and change substat roll weights?

**Comment by Birdulon** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/5397662/175484488-ed8c41fa-5566-480f-9dbb-3b1174b9e958.png)

**Reply by maxkleehu** [reply]
ty :D

**Reply by maxkleehu** [reply]
the weights add up to 10k like banners?

**Reply by Birdulon** [reply]
I think there's no cap

**Comment by OuliKuato** [comment]
> ![image](https://user-images.githubusercontent.com/5397662/175484488-ed8c41fa-5566-480f-9dbb-3b1174b9e958.png)
> 

where is the file? i cant find it in github source code

**Reply by Birdulon** [reply]
https://github.com/Koko-boya/Grasscutter_Resources
